.github/workflows/test-gh.yml

```diff
@@ -29,40 +30,53 @@ jobs:
         path: src/github.com/${{ github.repository }}
         fetch-depth: 0
     - name: Run Tests
+      env:
+        DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
@pivotaljohn what account are we including creds for?

Reply:
Today, it's "k8slt".

Reply:
Updated so that the username is in plaintext. 👍🏻